src/shared/buttons/BlueLinkButton.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const LinkStyled = styled.span`
+  height: 20px;
+  width: 118px;
+  color: #004699;
+  font-size: 16px;
+  font-weight: 900;
+  line-height: 20px;
+`;
+
+const BlueLinkButton = props => {
```

Comment:
`LinkButton` is een betere omschrijving voor nu toch? Zolang alles blauw is dan is het onderscheid niet echt nodig